src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/uc/UcTemplateContent.java

```diff
@@ -0,0 +1,197 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.writefinaldecision.uc;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.removeStart;
+import static org.apache.commons.lang3.StringUtils.startsWith;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.writefinaldecision.uc.scenarios.UcScenario;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.writefinaldecision.uc.scenarios.UcTemplateComponentId;
+import uk.gov.hmcts.reform.sscs.model.docassembly.Descriptor;
+import uk.gov.hmcts.reform.sscs.model.docassembly.Paragraph;
+import uk.gov.hmcts.reform.sscs.model.docassembly.WriteFinalDecisionTemplateBody;
+import uk.gov.hmcts.reform.sscs.model.docassembly.WriteFinalDecisionTemplateContent;
+
+public abstract class UcTemplateContent extends WriteFinalDecisionTemplateContent {
+
+    protected static DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+
+    public String getAllowedOrRefusedSentence(boolean allowed) {
+        return "The appeal is " + (allowed ? "allowed" : "refused") + ".";
+    }
```

Comment:
Maybe one to think about, but this sort of generic content that could be refused no matter what the benefit type, i wonder if we could move somewhere?